ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -7471,3 +7471,44 @@ def test_torch_instance_div_(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# bitwise right shift
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_right_shift",
+    dtype_and_x=helpers.dtype_and_values(
```

Comment:
It is recommended to have the two arrays of the same type. Use `shared_dtype` for that

Reply:
Hi @ZiadAmerr, added the `shared_dtype`.